### ISSUE TITLE
[ios] Missing VoiceOver button labels

### ISSUE THREAD

**Issue by tyrylu** [tailscale/tailscale#4974]
### What is the issue?

When using the app with VoiceOver, the more actions button and the active button/whatever it is does not have a label for the iOS screen reader and the heuristics must kick in. Adding them should be relatively straightforward, though.

### Steps to reproduce

1. Activate VoiceOver
2. Open the app
3. Browse it using the screen reader

### OS

iOS

### OS version

15.5

### Tailscale version

1.26.1

**Comment by mihaip** [comment]
Also tracked on the corp side as tailscale/corp#5799

**Comment by agottardo** [comment]
We have added a lot of accessibility labels and hints lately, providing a better experience for VoiceOver users. The majority will be landing in the next app update, 1.50. Once you have tried this build, please mention me here if you notice specific UI components that require improvements.
